build-system/configuring-vxworks-6_x.md

```diff
@@ -1,5 +1,8 @@
 # Configuring vxWorks 6.x
 
+```{tags} user, advanced
```

Comment:
consider making that "developer, advanced"
as users don't handle the central system configuration